volume/drivers/aws/aws_test.go

```diff
@@ -4,19 +4,48 @@ import (
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/opsworks"
 	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/volume"
 	"github.com/libopenstorage/openstorage/volume/drivers/test"
+	"github.com/stretchr/testify/require"
 )
 
+func testRemoveTags(t *testing.T, driver volume.VolumeDriver) {
+	d := driver.(*Driver)
+	// Create volume with labels
+	sz := int64(1)
+	voltype := opsworks.VolumeTypeIo1
+	ec2Vol := &ec2.Volume{
+		AvailabilityZone: &d.md.zone,
+		VolumeType:       &voltype,
+		Size:             &sz,
+	}
+	labelNames := []string{"label1", "label2"}
+	labels := make(map[string]string)
+	for _, name := range labelNames {
+		labels[name] = name
+	}
+	vol, err := d.ops.Create(ec2Vol, labels)
```

Comment:
Does this means that this test has to be run on aws?

Reply:
yes. The test is run only if it determines that it can be run.